internal/cli/internal/command/command.go

```diff
@@ -426,3 +426,18 @@ func LoadAppNameIfPresent(ctx context.Context) (context.Context, error) {
 
 	return app.WithName(ctx, name), nil
 }
+
+// RequireRegion is a Preparer which makes sure the user has selected a region via command line arguments,
+func RequireRegion(ctx context.Context) (context.Context, error) {
+	ctx, err := LoadAppConfigIfPresent(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	region := flag.GetRegion(ctx)
```

Comment:
`flag.GetRegion` tells you whether the region flag has been specified. If not, you should consult with what `Region` in `config.FromContext(ctx)` has in it.